client/src/lib/loadFromLocal.js

```diff
@@ -0,0 +1,7 @@
+export default function loadFromLocal(key) {
```

Comment:
Seems like you created two loadFromLocal functions - one in lib/loadFromLocal, one in lib/localStorage. If there wasn't a deeper reason behind it, you could delete this js-document here.

Reply:
Before I implemented the local storage custom hook, I had two seperate files for "load" and "save". I intended to delete those two, which were no longer needed. Apparently it didnt happen for the "loadFromLocal" file.